src/shared/CustomInput/CustomInput.tsx

```diff
@@ -65,6 +67,11 @@ export const CustomInput = forwardRef<HTMLInputElement, Props>(
                         headerTitle="Select asset"
                         onChange={onAssetValueChange}
                     />
+                    <div className={styles.empty_container}>
+                        {isLoading && !isInputEnabled ? (
+                            <div className={styles.loader_spinner} />
+                        ) : null}
```

Comment:
how `isInputEnabled` is related to loading state?

Reply:
if you need to display loading state only on output asset - you pass `isLoading` prop only to output asset component (what you are already did)